### PR TITLE
Feature fix 3644

### LINKED ISSUE #3644: Remove attachment Text BBcode into Post
Remove attachment Text BBcode into Post

When Edit a post can delete or add attachment  and can -Insert into a Post -

but When Remove attachment... The BBCode  - Insert into a Post -  Still in the Post!

Reproduction steps
1. Create a post with attachment.
2. Add Attachment.
3. - Insert into a Post -  Buton BBCode
4. Post Reply.

Issue:
When Delete the Attachment  The BBCode -Insert into a Post -  Still in the Post!

- It would be good to delete the BBcode from the Post too!

it's just a suggestion... The Software its Amazing!

Thank you!  :)

Original thread: [Remove attachment Text BBcode into Post](https://community.mybb.com/thread-222217.html)

### PR DESCRIPTION
**Pull request by veryard**
Resolves #3644 

Replaces #3629 - Couldn't find branch

### CODE REVIEW

**Comment by veryard** [comment]
Sorry for the commits, cleaned it up so it didn't have other changes from other branches.

**Comment by euantorano** [comment]
No problem, they'll get squashed when we merge anyway 😊

File: jscripts/post.js [hunk 1]
```diff
@@ -131,6 +131,19 @@ var Post = {
 							{
 								$('#attachment_'+aid).hide(500, function()
 								{
+                                    var instance = MyBBEditor;
+
+                                    if(typeof MyBBEditor == 'undefined') {
+                                        instance = $('#message').sceditor('instance');
+                                    }
```
Comment (effone) [inline jscripts/post.js]: Just some indentation issue here; otherwise looks perfect.

Comment (veryard) [reply jscripts/post.js]: Updated.

**Pull request merged**